SU-SWS/stanford_profile
# D8CORE-1828 Enable google tag manager

# READY FOR REVIEW

# Summary
- Enable google tag manager
- ignore GTM container config entiies
- Use config override to disable GTM containers on non-prod environments.
- Allow config changes with config readonly https://github.com/SU-SWS/acsf-cardinalsites/pull/173

# Need Review By (Date)
- 7/22

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-1828`
1. do a fresh install or `blt drupal:update`
1. add a container `/admin/config/system/google-tag`
1. verify its disabled by viewing the page source when incognito or using `drush cget google_tag.container.[container_name] --include-overridden`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
